RC: Clarify that monthly network cap applies over all databases in the plan (#996)
redis/docs: content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md

|---|---|---|---|---|---|---|
| **Concurrent<br/>connections<br/>per database** | 30 | 256 | 1024 | 2500 | 5000 | 10000 |
| **CIDR<br/> allow rules** | 1 | 4 | 4-8 | 4-8 | 4-16 | 4-32 |
| **Monthly<br/> total network<br/> bandwidth<sup>[2](#table-note-2)</sup>** | 5&nbsp;GB | 100&nbsp;GB | 200&nbsp;GB | 400&nbsp;GB | 800&nbsp;GB | 2000&nbsp;GB |
| **Maximum<br/> throughput<sup>[3](#table-note-3)</sup>** | 100&nbsp;ops/sec | 1000&nbsp;ops/sec | 2000&nbsp;ops/sec | 4000&nbsp;ops/sec | 8000&nbsp;ops/sec | 16000&nbsp;ops/sec |
{{</table-scrollable>}}

1. <a name="table-note-1" style="display: block; height: 80px; margin-top: -80px;"></a> Database size includes replication where applicable. See [High availability cost impact]({{< relref "/operate/rc/databases/configuration/high-availability#dataset-size" >}}) for more information.

> 2. <a name="table-note-2" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies over all databases in the plan.

Reviewer: ```suggestion
2. <a name="table-note-2" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies to all databases in the plan.
```

Reviewer: I like it!

| **Concurrent<br/>connections<br/>per database**                                   | 30               | 256              | 256               | 512               | 1024              | 2500              | 5000              | 10000              |
| **Monthly<br/> total network<br/> bandwidth**<sup>[2](#table-note-2-legacy)</sup> | 5&nbsp;GB        | 50&nbsp;GB       | 100&nbsp;GB       | 150&nbsp;GB       | 200&nbsp;GB       | 400&nbsp;GB       | 800&nbsp;GB       | 2000&nbsp;GB       |
| **Maximum<br/> throughput<sup>[3](#table-note-3-legacy)</sup>**                                                       | 100&nbsp;ops/sec | 500&nbsp;ops/sec | 1000&nbsp;ops/sec | 1500&nbsp;ops/sec | 2000&nbsp;ops/sec | 4000&nbsp;ops/sec | 8000&nbsp;ops/sec | 16000&nbsp;ops/sec |
{{</table-scrollable>}}

1. <a name="table-note-1-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> Database size includes replication. See [High availability cost impact]({{< relref "/operate/rc/databases/configuration/high-availability#dataset-size" >}}) for more information.

> 2. <a name="table-note-2-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies over all databases in the plan.

Reviewer: ```suggestion
2. <a name="table-note-2-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies to all databases in the plan.
```

Author: ```suggestion
2. <a name="table-note-2-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies to the entire plan. All databases in the plan share the allocated bandwidth limit.
```
